docs/DRONECAN.md

```diff
@@ -0,0 +1,38 @@
+# mLRS Documentation: DroneCAN for ArduPilot Systems #
+
+([back to main page](../README.md))
+
+DroneCAN can be utilized for MAVLink and RC with mLRS instead of traditional serial communication if both the flight controller and receiver have the hardware necessary to support it.
+
+## Wiring
+
+TBD
+
```

Comment:
maybe a chapter before

## mLRS Receiver Settings

With a CAN-enabled firmware flashed, the receiver will output the RC channels data on the CAN bus without any further settings.

In order to enable also the serial (MAVLink) communication via DroneCAN, one needs to set "Rx Serial Port = can".

Reply:
Added